src/components/biggive-tabbed-content/biggive-tabbed-content.tsx

```diff
@@ -1,24 +1,125 @@
-import { Component, Prop, h } from '@stencil/core';
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
 
 @Component({
   tag: 'biggive-tabbed-content',
   styleUrl: 'biggive-tabbed-content.scss',
   shadow: true,
 })
 export class BiggiveTabbedContent {
-  /**
-   * Space below component
-   */
-  @Prop() spaceBelow: number = 0;
+  @Element() host: HTMLBiggiveTabbedContentElement;
 
-  /**
-   * Default text colour
-   */
-  @Prop() defaultTextColour: string = 'black';
+  @Prop() spaceBelow: spacingOption = 0;
+
+  @Prop() textColour: brandColour = 'black';
+
+  @Prop() selectedTextColour: brandColour = 'primary';
+
+  @Prop() navigationHighlightColour: brandColour = 'grey-medium';
+
+  @Prop() selectedNavigationHighlightColour: brandColour = 'primary';
+
+  @Prop() buttonBackgroundColour: brandColour = 'white';
+
+  @Prop() buttonIconColour: brandColour = 'primary';
+
+  currentTab = 0;
+  children: Array<any> = [];
```

Comment:
as above, do we need to use the `any` type here?

Reply:
Done